authors/settings/base.py

```diff
@@ -84,18 +85,27 @@
 # Password validation
 # https://docs.djangoproject.com/en/1.11/ref/settings/#auth-password-validators
 
+USER_ATTRIBUTE_SIMILARITY_VALIDATOR = \
```

Comment:
Thanks for the good work. If you don't mind me asking, what is the functionality of the lines below that were added in this file

Reply:
@missvicki I touched this file so they were pep8 issues I was just resolving those issues